app/authentication/authenticator.py

```diff
@@ -22,19 +22,19 @@
 
 @login_manager.user_loader
 def user_loader(user_id):
-    logger.debug('loading user', user_id=user_id)
+    logger.info('loading user', user_id=user_id)
```

Comment:
We should either leave these as debug, or review the usefulness of these log statements rather that just change them all to info. It would be better to have logging on the un-happy path so we log when something un-expected happens.

Also, by changing all of these statements to debug we will be adding a lot of extra logging to every request in production - from 2 to 8 log lines.

Reply:
We don't (or shouldn't be) manually parsing log lines given we use the GCP logging service. This negates the 'can't see the wood for the trees' problem. Even when we log the unhappy path, it can still be useful to see what steps were taken up to that point.

I couldn't say whether increasing from 2 to 8 is a lot of extra logging without running some performance tests (which I'll carry out and post results here - it would be interesting to see what impact the extra logs have in any case).

Reply:
I agree with what you're saying, just maybe not on the `census-rehearsal` branch.

Reply:
Does this refer to all the debug -> info changes or just the ones in this module?

Reply:
All the debug -> info changes.